test/common/DaemonConfiguration.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace common
+{
+    using System;
+    using System.IO;
+
+    public class DaemonConfiguration
+    {
+        const string ConfigYamlFile = @"C:\ProgramData\iotedge\config.yaml";
+
+        YamlDocument config;
+
+        public DaemonConfiguration()
+        {
+            string contents = File.ReadAllText(ConfigYamlFile);
+            this.config = new YamlDocument(contents);
+        }
+
+        public void AddHttpsProxy(string proxyUrl)
```

Comment:
Use Uri type

Reply:
Added validation to proxy command line argument. Since this is test code I want to limit the amount of extra checks internally.